Issue51 - Make basic example compile again

bjorgan opened the pull request:
Fixes issue #51. Examples can be compiled again.

Also changed the printing in the basic example a bit, by clearing the screen and printing on top of the terminal (making it seem like numbers are updated instead of being printed row by row once every second). Makefile was also exchanged by CMakeLists.txt-files. Examples was added to travis build so that broken examples don't have to happen again.

bjorgan committed:
Add cmakelists for basic example, make basic example compile.

bjorgan committed:
Add CMakeLists for setrise example.

bjorgan committed:
Make travis build examples and add common CMakeLists to examples subdirectory.

ryeng commented:
Thanks for implementing this!

The change looks gode, but can we squash some of the commits to get a nice and clean commit history with logical steps? E.g., the verbosity of Travis is adjusted up and down in various patches in this branch.

bjorgan commented:
@ryeng: I've squashed the travis-related commits to one commit. Also rebased on master and changed call to predict_parse_tle() in example accordingly.
